docs/workflows/upload.rst

```diff
@@ -1,55 +0,0 @@
-Upload and Manage Content
-=========================
-
-Create a repository
--------------------
-
-If you don't already have a repository, create one::
-
-    $ http POST $BASE_ADDR/pulp/api/v3/repositories/npm/npm/ name=foo
-
-Response::
-
-    {
-        "pulp_href": "http://localhost:24817/pulp/api/v3/repositories/npm/npm/9b19ceb7-11e1-4309-9f97-bcbab2ae38b6/",
-        ...
-    }
-
-
-Upload a file to Pulp
```

Comment:
We are missing the workflows from the old `upload.rst`, right? Not sure if they are still valid, considering that artifact upload should not be allowed anymore.